noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -1796,64 +2021,15 @@
-        <queries>
-            <info>
-                <![CDATA[
-                    SELECT s.systemid series_system_id,
-                        COALESCE(series.val, 0) number_of_series_gradings,
-                        COALESCE(class.val, 0) number_of_class_gradings,
-                        COALESCE(file.val, 0) number_of_file_gradings,
-                        COALESCE(item.val, 0) number_of_record_gradings,
-                        COALESCE(document.val, 0) number_of_document_descriptions_gradings
-                    FROM arkivstruktur.arkivdel s
-                    LEFT JOIN
-                    (
-                        SELECT g._ref_arkivdel series_system_id, COUNT(*) val
-                        FROM arkivstruktur.gradering g
-                        GROUP BY g._ref_arkivdel
-                    ) series ON series.series_system_id = s.systemid
-                    LEFT JOIN
-                    (
-                        SELECT ks._ref_arkivdel series_system_id, COUNT(*) val
-                        FROM arkivstruktur.gradering g
-                        JOIN arkivstruktur.klasse k ON k.systemid = g._ref_klasse
-                        JOIN arkivstruktur.klassifikasjonssystem ks ON ks.systemid = k._ref_klassifikasjonssystem
-                        GROUP BY ks._ref_arkivdel
-                    ) class ON class.series_system_id = s.systemid
-                    LEFT JOIN
-                    (
-                        SELECT f._ref_arkivdel series_system_id, COUNT(*) val
-                        FROM arkivstruktur.gradering g
-                        JOIN arkivstruktur.mappe f ON f.systemid = g._ref_mappe
-                        GROUP BY f._ref_arkivdel
-                    ) file ON file.series_system_id = s.systemid
-                    LEFT JOIN
-                    (
-                        SELECT r._ref_arkivdel series_system_id, COUNT(*) val
-                        FROM arkivstruktur.gradering g
-                        JOIN arkivstruktur.registrering r ON r.systemid = g._ref_registrering
-                        GROUP BY r._ref_arkivdel
-                    ) item ON item.series_system_id = s.systemid
-                    LEFT JOIN
-                    (
-                        SELECT r._ref_arkivdel series_system_id, COUNT(*) val
-                        FROM arkivstruktur.gradering g
-                        JOIN arkivstruktur.dokumentbeskrivelse d ON d.systemid = g._ref_dokumentbeskrivelse
-                        JOIN arkivstruktur.registrering r ON r.systemid = d._ref_registrering
-                        GROUP BY r._ref_arkivdel
-                    ) document ON document.series_system_id = s.systemid;
-                ]]>
-            </info>
-        </queries>
-    </rule>
+    </test>
 
-    <rule>
+    <test id="AST20">
         <title>Disposal decision count</title>
+        <description>
+            Provides information about the number of Disposal decisions related to Series, Classes,
```

Comment:
`Provides information about the number of disposal decisions related to series, classes, files, records, and document descriptions, and tests whether the corresponding number in arkivuttrekk.xml (inneholderDokumenterSomSkalKasseres) is correct.`